packages/js/src/manage-ai-consent-button/initialize.js

```diff
@@ -0,0 +1,82 @@
+import { useSelect } from "@wordpress/data";
+import domReady from "@wordpress/dom-ready";
+import { Fragment, render, useCallback, useRef } from "@wordpress/element";
+import { __ } from "@wordpress/i18n";
+import { Modal, useToggleState } from "@yoast/ui-library";
+import classNames from "classnames";
+import { get } from "lodash";
+import { HAS_AI_GENERATOR_CONSENT_NAME, PLUGIN_URL_NAME } from "../shared-admin/store";
+import { GrantConsent } from "./components/grant-consent";
+import { RevokeConsent } from "./components/revoke-consent";
+import { STORE_NAME_CONSENT_BUTTON } from "./constants";
+import { registerStore } from "./store";
+
+// Yoast SEO free dependencies.
+const { setTextdomainL10n } = window.yoast.editorModules.helpers.i18n;
+
+// Register the Premium translations to the domain.
+setTextdomainL10n( "wordpress-seo-premium", "wpseoPremiumJSL10n" );
+
+domReady( () => {
+	const WISTIA_EMBED_PERMISSION_NAME = get( window, "yoast.editorModules.sharedAdmin.store.WISTIA_EMBED_PERMISSION_NAME", "wistiaEmbedPermission" );
```

Comment:
Let's not use the editor modules in Free itself. We can just use the code directly here with proper imports.

Reply:
Good point, there is no video there anymore.
Should we just remove the whole thing from the store?